find_gons.py

```diff
@@ -31,4 +24,19 @@ def request_api(go_term):
     if result.ok:
```

Comment:
It would be useful to add a timeout to request and check the status to distinguish a network failure from 'not found' answer.

Reply:
I have updated it, Please check it out. Thanks